python/servo/bootstrap_commands.py

```diff
@@ -126,6 +131,79 @@ def env(self):
         else:
             print("export LD_LIBRARY_PATH=%s" % env["LD_LIBRARY_PATH"])
 
+    @Command('bootstrap-msvc',
+             description='Download the Windows MSVC dependencies',
+             category='bootstrap')
+    @CommandArgument('--force', '-f',
+                     action='store_true',
+                     help='Force download even if a copy already exists')
+    def bootstrap_msvc(self, force=False):
+        # Don't download MSVC dependencies on non-Windows platform
+        if not is_windows():
+            return print("Unsupported platform.")
+
+        msvc_deps_dir = path.join(self.context.sharedir, "msvc-dependencies")
+        msvc_deps_url = "https://dl.dropboxusercontent.com/u/25971865/msvc-deps/"
```

Comment:
We will definitely want to move these to one of the Servo S3 buckets. I like that you've versioned the files, which is something we've definitely found a need for in the past.